front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -438,7 +420,7 @@ export default function AssistantBuilder({
         }
         valid = false;
       } else if (!(await assistantHandleIsAvailable(builderState.handle))) {
-        setAssistantHandleError("Assistant handle is already taken");
+        setAssistantHandleError("\u26a0 This handle is already taken");
```

Comment:
Do we really want the unicode char vs an icon ?

Reply:
In this specific case I don't think there is a benefit to use an icon. Are unicode chars an issue? (this one is standard)